src/game/Globals/ObjectMgr.cpp

```diff
@@ -3877,7 +3877,7 @@ void ObjectMgr::LoadGroups()
                  "(SELECT COUNT(*) FROM character_instance WHERE guid = group_instance.leaderGuid AND instance = group_instance.instance AND permanent = 1 LIMIT 1), "
                  // 7              8
                  " groups.groupId, instance.encountersMask "
-                 "FROM group_instance LEFT JOIN instance ON instance = id LEFT JOIN groups ON groups.leaderGUID = group_instance.leaderGUID ORDER BY leaderGuid"
+                 "FROM group_instance LEFT JOIN instance ON instance = id LEFT JOIN wotlkcharacters.groups ON groups.leaderGUID = group_instance.leaderGUID ORDER BY leaderGuid"
```

Comment:
i guess wotlkcharacters is your local database name, i can be different for the others